src/modifiers.rs

```diff
@@ -113,7 +113,7 @@ pub struct Redirect(pub Url);
 impl Modifier<Response> for Redirect {
     fn modify(self, res: &mut Response) {
         let Redirect(url) = self;
-        res.headers.set(headers::Location(format!("{:?}", url)));
+        res.headers.set(headers::Location(format!("{}", url)));
```

Comment:
@s-panferov, Url doesn't have a to_string method as far as I know. If it had/has, to_string would be better.